### PR TITLE
feat(playwright-test): Add a multi-page example

### LINKED ISSUE #135: Create multi-page TS example for Playwright w/config
Create a multi-page test in playwright w/config in typescript that already has Watcher integrated into it. When run, the test should result in a few accessibility errors. Verify these results make it up to developer hub and are expected and consistent before closing this ticket.

### PR DESCRIPTION
**Pull request by stephenmathieson**
Closes #135

### CODE REVIEW

File: playwright-test/multi-page/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "multi-page",
+  "private": true,
+  "scripts": {
+    "test": "playwright test"
+  },
+  "devDependencies": {
+    "@axe-core/watcher": "^2.0.0-next.099d5645",
```
Comment (stephenmathieson) [inline playwright-test/multi-page/package.json:8]: Support for `@playwright/test` won't be in a stable version until v2.1.0. We **must** use the `@next` channel.

File: playwright-test/multi-page/tests/forgot-password.spec.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import type { ElementHandle } from '@playwright/test'
+import { test, expect } from './fixtures'
+
+test.describe('Forgot password page', () => {
```
Comment (Zidious) [inline playwright-test/multi-page/tests/forgot-password.spec.ts:4]: Should we added the `afterEach` to the tests too? I am just thinking once we add it to the docs and they come here to reference and wonder why we don't have the `afterEach` unless we communicate that it's we do it as part of `playwrightTest`

**Review by Zidious — APPROVED** [review]
One question inline - otherwise LGTM

File: playwright-test/multi-page/tests/forgot-password.spec.ts (hunk 2 above)
Comment (stephenmathieson) [reply playwright-test/multi-page/tests/forgot-password.spec.ts:4]: No, [`@playwright/test` does it automagically](https://github.com/dequelabs/jazzband/blob/develop/integrations/javascript/src/playwrightTest.ts#L76-L78). It is not needed/included in docs.

**Pull request merged**